src/main/java/frc/robot/RobotContainer.java

```diff
@@ -181,18 +187,107 @@ public Command getTeleopDriveCommand() {
     return robotDrive.getDriveCommand(driverController);
   }
 
+  /** Setup the options for the Autonomous mode command chooser. */
+  private void setupAutoChooser() {
+
+    autoChooser.setDefaultOption("Nothing", "Nothing");
+    autoChooser.addOption("Taxi", "DriveStraight");
+    autoChooser.addOption("Shoot", "Shoot");
```

Comment:
Please add "and Stop" to the label to make it clear that there is nothing after that. This will also clear up the new issue introduced.

Reply:
These were implemented by other issues. I merged the week3 branch into issue 73 before I created the PR.  I'll talk to Jay about it on Wednesday.